src/DataTransferObject/AssignmentDto.php

```diff
@@ -36,7 +38,7 @@ class AssignmentDto
     /** @var int|null */
     private $userId;
 
-    public function __construct(string $state, int $lineItemId, string $username, int $userId = null)
+    public function __construct(string $state, UuidV6 $lineItemId, string $username, int $userId = null)
```

Comment:
Not that it really matters or is in any standards, but:
IMO - Object arguments should go first, as it improved readability a bit